site/docs/v1/tech/apis/_form-response-body.adoc

```diff
@@ -26,6 +26,8 @@ An ordered list of Form Field Ids assigned to this step.
 The form type. The possible values are:
 +
 * `registration`
+* `adminRegistration`
+* `adminUser`
```

Comment:
1. Add Since info, see other examples in the docs for adding this as a suffix to these values. 
2. Each type should also reflect the same detail from the request.